modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/layouts/BaseLayout/index.js

```diff
@@ -31,26 +31,46 @@ const PAGE_SKELETON_LAYOUT = {
 
 const Layout = () => {
 	const location = useLocation();
+	const [, ...currentPath] = location.pathname.split('/').filter(Boolean);
 
 	const [
 		{project, sessionId, subscriptionGroups, userAccount},
 	] = useCustomerPortal();
 
 	const getCurrentPage = useCallback(() => {
-		const [, ...currentPath] = location.pathname.split('/').filter(Boolean);
-
 		return currentPath.length
 			? currentPath.slice(-1)[0]
 			: PAGE_TYPES.overview;
-	}, [location]);
+	}, [currentPath]);
+
+	const getCurrentProduct = () => {
+		const activationKey = 'activation';
+
+		const isProduct = !!currentPath.find((path) => path === activationKey);
+
+		if (isProduct) {
+			const [, ...productType] = currentPath;
+
+			return productType.join('_');
+		}
+
+		return null;
+	};
 
 	const hasProjectContact = getCurrentPage() === PAGE_TYPES.overview;
 
-	const hasQuickLinksPanel = getCurrentPage() !== PAGE_TYPES.teamMembers;
+	const hasQuickLinksPanel =
+		getCurrentPage() !== PAGE_TYPES.teamMembers &&
+		getCurrentProduct() !== PAGE_TYPES.dxp_new;
+
+	const hasSideMenu = getCurrentProduct() !== PAGE_TYPES.dxp_new;
```

Comment:
Should be good pass the return of function to variables and using them to check.
```
const currentPage = getCurrentPage();
const currentProduct  = getCurrentProduct();

const hasQuickLinksPanel =
		currentPage !== PAGE_TYPES.teamMembers &&
		currentProduct !== PAGE_TYPES.dxp_new;
```